### PR TITLE
feat: update default page style

### LINKED ISSUE #1371: Remove title padding top
https://affine-next.vercel.app/preview/Welcome-to-AFFiNE-Alpha-Downhills

6875DD4F-7DAF-4E96-B417-400E57837569

### PR DESCRIPTION
**Pull request by zqran**
close: #1371

### CODE REVIEW

**Review by himself65 — APPROVED** [review]

**Comment by himself65** [comment]
LGTM but Dont merge until next AFFiNE has land

**Review by doodlewind — APPROVED** [review]
Should be good to merge since AFFiNE next is landed, /cc @Himself65

**Pull request merged**
